### PR TITLE
Deploy Etiquette section

### PR DESCRIPTION
**Pull request by jessb0t**
@georgebuzzell 
Etiquette section (which includes Zenhub page in technical-docs) is ready for official PR! 😄

### CODE REVIEW

File: docs/etiquette/programming-standards.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
```
Comment (georgebuzzell) [inline docs/etiquette/programming-standards.md:1]: @F-said and @jessb0t I think this programming standards doc is a really, really great start and good for pushing to main as is. That said, we will want to refine/expand this doc further with additional details and examples to make it easier to understand by novices. but again, no need to hold up pushing to main as is!

**Review by georgebuzzell — CHANGES_REQUESTED** [review]

**Comment by georgebuzzell** [comment]
@jessb0t I think this looks really, really, really great! I only have very minor comments/edits. I marked this as "changes requested", but please note that once the minor comments are addressed I am happy to give a quick final approve and merge!!

File: docs/etiquette/naming-conventions.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+> `dev-stimuli-alice`
+
+Additionally, when Alice creates her new branch off `dev-stimuli`, Bob should also create a branch off `dev-stimuli` and call it:
+
+> `dev-stimuli-bob`
+
+(Note there are two exceptions to the above branching rules. The first is the wiki, which does not have a `dev` branch, and new branches are created directly off `main` according to the convention `[feature]-[yourname]`. The second exception applies to external collaborators (that do not have push access to a given NDCLab repo) and therefore must make all contributions following a "fork and pull" model. For such external collaborators, all the same, branch naming rules apply, with the exception being that it is never necessary to create additional branches with the user's name, as all branches will, by definition, occur within a personal repository of the external collaborator.)
```
[the remark this answers is not in the document]
Comment (jessb0t) [reply docs/etiquette/naming-conventions.md:157]: Roger!

File: docs/technical-docs/zenhub.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+2. Download the ZenHub extension, which is currently available for Chrome and Firefox. You can do this directly from the browser extension store or using the [link available from ZenHub](https://help.zenhub.com/support/solutions/articles/43000507578-installing-the-zenhub-extension-for-cloud).
+3. During this process, you will need to authorize ZenHub to see your personal and organization repositories.
+4. After installing, you'll probably find yourself within the ZenHub app. Some lab members prefer to operate in this interface, but you will want to hop back over to GitHub to get your bearings first. On the same browser where you installed ZenHub, re-navigate to the NDCLab GitHub page. If you click on any repository, you should now see the "ZenHub" tab within that repository.
+
+![gh_zenhub-tab](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/technical/gh_zenhub-tab.png)
+
+## Tour of the Pipelines
+
+When a new workspace is created, it uses the ZenHub defaults. These are customized for each NDCLab workspace into the following pipelines.
```
[the remark this answers is not in the document]
Comment (jessb0t) [reply docs/technical-docs/zenhub.md:32]: @georgebuzzell The explanation you are looking for is on github-tempates.md as part of the setup of a new project.

For any given instruction, there are two possible locations on the wiki:
1. The technical page that provides a detailed overview of how to interact with the tool.
2. The workflow page that walks someone through a setup.

As you know, I am a strong advocate of avoiding duplications, which means that I was making a case-by-case decision on the best location.  For this particular case, my thought process was this: Most team members will be interacting with ZenHub **after** the pipelines are established.  So the key information for them is what lives in each pipeline and how to work with them (not how to set them up).  Therefore:
- the details of how pipelines should be set up was placed on the "technical" page,
- the instruction to actually match this setup during the creation of a new project was placed on the "workflow" page.

Here on the "technical" page, I attempted to provide a simple indication that the lab customizes the pipelines.  Do you think more clarity is needed?
> When a new workspace is created, it uses the ZenHub defaults. These are customized for each NDCLab workspace into the following pipelines.

File: docs/etiquette/naming-conventions.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+2. [General Naming Guidance](#general-naming-guidance)
+3. [Code](#code)
+4. [GitHub](#github)
+5. [REDCap](#redcap)
+6. [Helpful Resources](#helpful-resources)
+
+## Overarching Philosophy
+
+All naming conventions for the NDCLab are designed to be:
+* human-friendly (that is, they are descriptive and help us navigate all our projects)
```
[the remark this answers is not in the document]
Comment (jessb0t) [reply docs/etiquette/naming-conventions.md:19]: @georgebuzzell I have added BIDS and FAIR!

**Comment by jessb0t** [comment]
@georgebuzzell All comments above and changes committed addressed.  This is ready for your re-review!

File: docs/technical-docs/zenhub.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (georgebuzzell) [reply docs/technical-docs/zenhub.md:32]: @jessb0t I think changing "zenhub defaults" to "NDCLab Zenhub defaults" (and making that phrase a link to the technical page with further details would clarify things and also avoid duplication. Agree?

**Review by georgebuzzell — APPROVED** [review]
@jessb0t "approve with mionr revisions" :)

@jessb0t I think changing "zenhub defaults" to "NDCLab Zenhub defaults" (and making that phrase a link to the technical page with further details would clarify things and also avoid duplication. Agree?

File: docs/technical-docs/zenhub.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jessb0t) [reply docs/technical-docs/zenhub.md]: @georgebuzzell Agree.  Implemented!

**Pull request merged**